README.md

```diff
@@ -1,17 +1,72 @@
 # MergeBack Bot
 
-[![Docker Build Status](https://img.shields.io/docker/build/existenz/mergeback.svg?style=flat-square)](https://hub.docker.com/r/existenz/mergeback/builds/) [![Docker Pulls](https://img.shields.io/docker/pulls/existenz/mergeback.svg?style=flat-square)](https://hub.docker.com/r/existenz/mergeback/) [![License](https://img.shields.io/github/license/existenznl/docker-mergeback.svg?style=flat-square)](https://github.com/eXistenZNL/Docker-Mergeback/blob/master/LICENSE)
+[![Docker Build Status](https://img.shields.io/docker/build/existenz/merge-back.svg?style=flat-square)](https://hub.docker.com/r/existenz/mergeback/builds/) [![Docker Pulls](https://img.shields.io/docker/pulls/existenz/merge-back.svg?style=flat-square)](https://hub.docker.com/r/existenz/mergeback/) [![License](https://img.shields.io/github/license/existenznl/docker-mergeback.svg?style=flat-square)](https://github.com/eXistenZNL/Docker-Mergeback/blob/master/LICENSE)
 
 ## About
-A container that ships the merge-back command which can merge the production branch, e.g. master, back into all other branches, in a CI like GitLab-CI.
-Built upon [Alpine Linux](https://alpinelinux.org/), and comes with a single handy command that does all the heavy lifting.
+
+A container that ships the merge-back command which can merge the production branch, e.g. master, back into all other
+branches, in a CI like GitLab-CI. Built upon [Alpine Linux](https://alpinelinux.org/), and comes with a single handy
+command that does all the heavy lifting.
 
 ## Why
 
+When working with Continuous Deployment and Review Applications, you want to have all your branches up-to-date
+with what's live on production. This container ensures that all open branches are updated with your target branch.
+
 ## So how does it work?
 
+In this example we want to make sure our branch gets updated with everything that's merged into master.
+
+1) In your `.gitlab-ci.yml`, add the following stage:
```

Comment:
Could you please make this listing into a h3?

Reply:
Could you please add the text "Running this container can be done everywhere, but was mainly built to run inside a GitLab-CI pipeline. In the example below, we will show you how.

Reply:
:+1: